source/fuzz/transformation_access_chain.cpp

```diff
@@ -74,10 +74,7 @@ bool TransformationAccessChain::IsApplicable(
   switch (pointer->opcode()) {
     case SpvOpConstantNull:
     case SpvOpUndef:
-      // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/3185): When
-      //  fuzzing for real we would like an 'assert(false)' here.  But we also
-      //  want to be able to write negative unit tests.
-      return false;
+      assert(false && "Null or undefined pointer");
```

Comment:
Please change message to: "Access chains should not be created from null/undefined pointers".

And please add "return false" after the assertion.  That way in release mode - when there are no assertions - the function will return false, which is the right thing to do.

Reply:
Done